src/api/api.test.js

```diff
@@ -1,92 +1,91 @@
 // @flow
 /* eslint-env jest */
 import jestSetup from '../jestSetup'
-
 import moment from 'moment'
 
+import api from './'
 import {ApiError} from './errors'
 
 jestSetup.setup()
 jest.setTimeout(30 * 1000)
 
-let apiWithTestnet
-
-// TODO: delete if/else and restore tests
-if (moment().isAfter('2019-12-20')) {
-  test('heads up: this should already be fixed!', () => {
-    throw new Error()
-  })
-
-  describe('History API', () => {
-    // We have to mock config before importing api so it propagates in it
-    jest.setMock('react-native-config', {USE_TESTNET: true})
-    apiWithTestnet = require('./api')
-    it('can fetch history', async () => {
-      const addresses = [
-        '2cWKMJemoBakWtKxxsZpnEhs3ZWRf9tG3R9ReJX6UsAGiZP7PBpmutxYPRAakqEgMsK1g',
-      ]
-      const ts = moment('1970-01-01')
-
-      // We are async
-      expect.assertions(1)
-      const result = await apiWithTestnet.fetchNewTxHistory(ts, addresses)
-
-      expect(result.transactions[0]).toMatchSnapshot({
-        bestBlockNum: expect.any(Number),
-        lastUpdatedAt: expect.any(String), // these fields may change (e.g. after restarting a node)
-        submittedAt: expect.any(String),
-      })
+describe('History API', () => {
+  it('can fetch history', async () => {
```

Comment:
@v-almonacid the fix was to not use testnet?

Reply:
@nicarq yes, just use a mainnet wallet and modify anything else accordingly

Reply:
cool